lib/fluent/plugin/out_label_router.rb

```diff
@@ -117,6 +119,10 @@ def filter_select(match, metadata)
           unless match.namespaces.empty? || match.namespaces.include?(metadata[:namespace])
             return false
           end
+          # Break if list of namespaces is not empty and does not contain any entry that match actual namespace
+          unless match.namespaces_regex.empty? || match.namespaces_regex.any? { |pattern| Regexp.new(pattern).match?(metadata[:namespace]) }
```

Comment:
```suggestion
          unless match.namespaces_regex.empty? || match.namespaces_regex.any? { |pattern| Regexp.new(pattern).match?(metadata[:namespace_regex]) }
```
I'm from mobile so I'm might missing something

Reply:
But **metadata[:namespace_regex]** doesn't exist. 
**metadata[:namespace]** comes from **kubernetes.namespace_name** and is the source value we want to match with match.namespaces_regex